python/ray/includes/common.pxd

```diff
@@ -668,7 +668,7 @@ cdef extern from "ray/gcs/pubsub/gcs_pub_sub.h" nogil:
         CRayStatus Close()
 
 cdef extern from "ray/gcs/pubsub/gcs_pub_sub.h" namespace "ray::gcs" nogil:
-    c_vector[c_string] PythonGetLogBatchLines(const CLogBatch& log_batch)
+    c_vector[c_string] PythonGetLogBatchLines(CLogBatch log_batch)
```

Comment:
So just to make sure I grok the pattern.  The original contract was a constant pass by reference which was an immutable non copied version of the object.  The new contract is a non-constant pass by value, but we're avoiding that copy with the std::move... Is that all the right way to understand it?  If so, I'm not sure this swap trades well interface wise unless there's another gain that I'm not seeing (pardon my ignorance).